test/run-tests.js

```diff
@@ -208,7 +208,7 @@ function run_compress_tests() {
                     });
                     return false;
                 }
-                if (0 && test.reminify && !reminify(test.options, input_code, input_formatted, test.expect_stdout)) {
+                if (test.reminify && !reminify(test.options, input_code, input_formatted, test.expect_stdout)) {
```

Comment:
Please leave this out - I don't want to redo this for every `harmony` merge.

Reply:
Why release `uglify-es` at all then if every new `uglify-js` feature renders it unusable?

Reply:
I'm performing this merge on a best-effort basis, in the belief that it would be usable for the majority of use cases out there.

If this is not the case and requires more effort on my part, I would indeed stop making any future `uglify-es` releases.

Reply:
Thanks for your work on `uglify-es`. I appreciate you'd rather concentrate on `uglify-js`.

A `reminify` test failure is no different than a regular test failure. Disabling it would just mask future regressions. It would be best not to release `uglify-es` with known regressions and just let `harmony` diverge from `master`.

Reply:
Okay - in that case this is the last PR I would accept for `harmony` and it shall be frozen from now one.

Reply:
@alexlamsl any chance we get a quick bugfix release for `uglify-es` though? This regression hit many of our projects using `uglifyjs-webpack-plugin`.